Extension/lib/webrequest.js

```diff
@@ -147,6 +147,13 @@
             return;
         }
 
+        // And we don't need this check on newer than 58 chromes anymore
+        // https://github.com/AdguardTeam/AdguardBrowserExtension/issues/572
+        if (adguard.utils.browser.isChromeBrowser() &&

```

Comment:
No, we implemented hack with CSP because webRequest didn't intercept WebSocket requests. Now it can, so we don't need this logic.